### ISSUE TITLE
[Question] Load huggingface model for I2V finetuning

### ISSUE THREAD

**Issue by yjhong89** [hao-ai-lab/FastVideo#236]
### Motivation

Hi! 

I am currently working on training I2V-Hunyuan based on this repo.

To train I2V from T2V model, I modify `load_transformers` like this. 

```
    finetune_keys = {"img_in.proj.weight": 16}
    for finetune_key, keepdim in finetune_keys.items():
        for name, param in model.named_parameters():
            if name == finetune_key:
                new_entry = torch.zeros_like(param)
                new_entry[:,:keepdim] = state_dict[finetune_key]
                state_dict[finetune_key] = new_entry

    model.load_state_dict(state_dict, strict=True)
```

- so I can train I2V model with image latents and mask channel concatenation (my I2V model in_channel is 33 (16+16+1))

But this full fine-tuning results is quite bad, so I am trying to fine-tune with huggingface module (finetune_hunyuan_hf.sh)
- As diffusers module load checkpoint with `from_pretrained` method: https://github.com/hao-ai-lab/FastVideo/blob/554ee17de54b95432edd4465a65e75d809b4564f/fastvideo/utils/load.py#L251
- Is there any way to load in_channel 33 DiT model ?? Like first 16 channels from T2V model and other channels are zero-initialized.
- I am relatively newbie to diffusers so want to ask if there exists way to circumvent this diffuser loading problem.

Thanks!

**Comment by BrianChen1129** [comment]
Haven't tried I2V model yet. Also, currently we recommend using Lora to finetune.

**Comment by yjhong89** [comment]
Okay Thanks!
